spec/granite/integrators/find_or_spec.cr

```diff
@@ -0,0 +1,30 @@
+require "../../spec_helper"
+
+describe "find_or_create_by, find_or_initialize_by" do
+  Spec.before_each do
+    Parent.clear
```

Comment:
try removing the `before_each` and putting the `Parent.clear` in each test.  I'm interested to see if that fixes it.

Reply:
Remember`Spec.before_each` isn't scoped to the describe block, it runs before each test _application wide_.